src/Aloha/Twilio/Twilio.php

```diff
@@ -13,13 +13,24 @@ public function __construct($config) {
     public function message($to, $message, $from=null) {
         $twilio = $this->getTwilio();
         // Send SMS via Twilio SDK
-        return $twilio->account->sms_messages->create(
+        return $twilio->account->messages->sendMessage(
             is_null($from) ? $this->config['from'] : $from,
             $to,
             $message
         );
     }
 
+    public function messageMMS($to, $message, $media, $from=null) {
```

Comment:
Yes but that is in the Laravel 5 branch, this was not available in the Laravel 4 version that we use.